src/App.test.js

```diff
@@ -1,9 +1,17 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './App';
+import { shallow } from "enzyme";
+import React from "react";
 
-it('renders without crashing', () => {
-  const div = document.createElement('div');
-  ReactDOM.render(<App />, div);
-  ReactDOM.unmountComponentAtNode(div);
+import App from "./App";
+
+describe("App", () => {
+  it("renders NavBar, UserProfile, Posts, and Contacts", () => {
+    const wrapper = shallow(<App />);
+
+    expect(wrapper.children().map((child) => child.type().name)).toEqual([
```

Comment:
Good call. I like the `.children().map(...).toEqual([...])` approach so that it gives a single informative assertion for all the expected things... but +1 to directly asserting on the components.